pkg/skaffold/debug/transform.go

```diff
@@ -40,12 +40,14 @@ Kubernetes requires that containers within a podspec are uniquely named.
 For example, a pod with two containers named `microservice` and `adapter` may be:
 
   debug.cloud.google.com/config: '{
-    "microservice":{"devtools":9229,"runtime":"nodejs"},
-    "adapter":{"jdwp":5005,"runtime":"jvm"}
+    "microservice":{"artifactImage":"node-example","runtime":"nodejs","ports":{"devtools":9229}},
+    "adapter":{"artifactImage":"java-example","runtime":"jvm","ports":{"jdwp":5005}}
   }'
 
-Each configuration is itself a JSON object with a `runtime` field identifying the
-language runtime, and a set of runtime-specific fields describing connection information.
+Each configuration is itself a JSON object of type `ContainerDebugConfiguration`, with an
+`artifactName` recording the corresponding artifact's namne in the skaffold.yaml,
```

Comment:
s/artifactName/artifactImage/

Reply:
I should just rename this to `artifact`.  I'll do that separately; it'll be a bigger PR.